native/packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -166,15 +194,19 @@ const BpkBadge = (props: Props) => {
 
 BpkBadge.propTypes = {
   message: PropTypes.string.isRequired,
-  type: PropTypes.oneOf(Object.keys(BADGE_TYPES)),
   docked: PropTypes.oneOf(Object.keys(BADGE_DOCKED_TYPES)),
+  icons: PropTypes.arrayOf(PropTypes.oneOf(Object.keys(bpkIcons))),
+  iconSeparator: PropTypes.string,
   style: ViewPropTypes.style,
+  type: PropTypes.oneOf(Object.keys(BADGE_TYPES)),
 };
 
 BpkBadge.defaultProps = {
-  type: BADGE_TYPES.warning,
+  icons: [],
```

Comment:
I think the default should be `null` - no need to use an array if it isn't needed (remember to update prop types too if you do this).